internal/transport/grpchttp2/framer.go

```diff
@@ -141,10 +146,12 @@ type HeadersFrame struct {
 	HdrBlock []byte
 }
 
+// Header returns the 9 byte header for this frame.
```

Comment:
Same as the other comment. Here and elsewhere in this PR.

Reply:
Done